### PR TITLE
add BCERT svg icon

### PR DESCRIPTION
**Pull request by Blockcerts-Blockchain**
Add blockcerts BCERT token icon

### 📝 Description

Add logos for BCERT token - BCERT to ledger. Support for tokens was added to ledger live desktop already

Contract addresses (ethereum)-
BCERT - 0xbb1f919e70ecc55335548d76f5c15a9dc8f5808e

etherscan url: https://etherscan.io/token/0xbb1f919e70ecc55335548d76f5c15a9dc8f5808e

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Comment by Blockcerts-Blockchain** [comment]
Hi, any other action we can do to move this forward :) ?

**Comment by Blockcerts-Blockchain** [comment]
Still no reply on that ? How can we move this further ?

**Comment by LFBarreto** [comment]
hi :wave: in order to integrate BCERT icon to ledger live could you reduce the number of paths of your icon to a minimal number. keeping it as plain as possible. All the paths will be filled with the same color in the end so if you need to work with negative space you need to merge the paths together keeping the difference.

**Comment by Blockcerts-Blockchain** [comment]
> hi wave in order to integrate BCERT icon to ledger live could you reduce the number of paths of your icon to a minimal number. keeping it as plain as possible. All the paths will be filled with the same color in the end so if you need to work with negative space you need to merge the paths together keeping the difference.

Is there any software that could help doing that ? We're having hard time getting that solved.

File: libs/ledger-live-common/src/data/icons/svg/BCERT.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (LFBarreto) [inline libs/ledger-live-common/src/data/icons/svg/BCERT.svg:1]: you can remove this line :+1:

File: libs/ledger-live-common/src/data/icons/svg/BCERT.svg [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 360 360">
```
Comment (LFBarreto) [inline libs/ledger-live-common/src/data/icons/svg/BCERT.svg:2]: please remove `id` `data-name` `xmlns` attrs
And add `width="360" height="360"` to the attrs

File: libs/ledger-live-common/src/data/icons/svg/BCERT.svg [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  <path d="M183,9.87C91.47,9.87,17,84.34,17,175.87s74.47,166,166,166,166-74.47,166-166S274.53,9.87,183,9.87Zm.12,318.39c-84.5,0-153.25-68.74-153.25-153.24S98.62,21.77,183.12,21.77s153.23,68.75,153.23,153.25-68.74,153.24-153.23,153.24Z"/>
```
Comment (LFBarreto) [inline libs/ledger-live-common/src/data/icons/svg/BCERT.svg:3]: Please add a `fill` property with the color of your choice to all the `path`

File: libs/ledger-live-common/src/data/icons/svg/BCERT.svg [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  <g>
```
Comment (LFBarreto) [inline libs/ledger-live-common/src/data/icons/svg/BCERT.svg:4]: you can remove all the group tags `g` as they are unused and non necessary here.

**Review by LFBarreto — CHANGES_REQUESTED** [review]
Some changes are needed and then we can merge :+1:

**Comment by Blockcerts-Blockchain** [comment]
All done what expected, thanks for the clarification.

**Review by mlegall-ledger — APPROVED** [review]

**Comment by mlegall-ledger** [comment]
Isn't a changeset required before merging ?

**Comment by Justkant** [comment]
Bypassing branch protections, the tests are failing because the runners are different for a fork

**Pull request merged**